lib/mix/tasks/rclex_docker.ex

```diff
@@ -109,21 +112,7 @@ defmodule Mix.Tasks.RclexDocker do
     "#{ros_distribution}-ex#{elixir_version}-otp#{erlang_otp_version}"
   end
 
-  # TODO: implement
   def docker_command_exists?() do
-    true
-  end
-
-  @doc """
-  Hello world.
-
-  ## Examples
-
-      iex> RclexDocker.hello()
-      :world
-
-  """
-  def hello do
-    :world
+    not is_nil(System.find_executable("docker"))
```

Comment:
System.find_executable returns the path when exists, returns nil when doesn't exist.